packages.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta7" />
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00085" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.16-prerelease" />
```

Comment:
Yes and no. It's unrelated to the fix, but I think it's needed as the compiler won't build from scratch without it. Seems the old package is no longer available on nuget. It's in a seperate commit.